### ISSUE TITLE
Get stuck on fetching origin

### ISSUE THREAD

**Issue by ysc12451** [desktop/desktop#17620]
### The problem

I am using Github desktop on my windows desktop. First I tried to pull some files with colon in their names, but I failed since windows system does not allow colons in the file names. Then I use my WSL to pull these files, it worked. Then I changed their local names to legal names in windows system and tried to push them to the github repo. After this push, when I open github desktop, it gets stuck on "fetching origin" forever. The file names online are all changed, so that push was successful. But now since it gets stuck, I cannot do anything to this repo in github desktop

### Release version

Version 3.3.4 (x64)

### Operating system

Windows 11

### Steps to reproduce the behavior

open this repo
find there is a "never fetch"
click "fetch origin"
get stuck forever

### Screenshots

![1d20f3c7c60e95b6e6c461c6aa70e96](https://github.com/desktop/desktop/assets/103322258/9a434467-4d74-4e67-9323-84f000ab7fd6)

**Comment by sergiou87** [comment]
@ysc12451  Could you upload the log file from GitHub Desktop so that I could get some more information about this error? To access the log files go to the file menu in GitHub Desktop and select Help > Show Logs.
The log files are created daily -- please upload a log file as an attachment from a day where you experienced the issue.

**Comment by ysc12451** [comment]
Thanks so much for your help! I put the log here.

[2023-10-24.desktop.production.log](https://github.com/desktop/desktop/files/13168250/2023-10-24.desktop.production.log)

**Comment by steveward** [comment]
Thanks for the log file! It looks like there are some characters in the filename that Windows doesn't like:

```
stderr:
error: invalid path 'assignment2_release/logs/exp_2023-10-23 21:14:50/data_top1_accuracy_train/events.out.tfevents.1698114061.Mingyis-Book.1025.1'
error: invalid path 'assignment2_release/logs/exp_2023-10-23 21:14:50/data_top1_accuracy_val/events.out.tfevents.1698114089.Mingyis-Book.1025.3'
error: invalid path 'assignment2_release/logs/exp_2023-10-23 21:14:50/data_top5_accuracy_train/events.out.tfevents.1698114061.Mingyis-Book.1025.2'
error: invalid path 'assignment2_release/logs/exp_2023-10-23 21:14:50/data_top5_accuracy_val/events.out.tfevents.1698114089.Mingyis-Book.1025.4'
error: invalid path 'assignment2_release/logs/exp_2023-10-23 21:14:50/events.out.tfevents.1698113690.Mingyis-Book.1025.0'
error: invalid path 'assignment2_release/logs/exp_2023-10-23 21:14:50/models/checkpoint.pth.tar'
error: invalid path 'assignment2_release/logs/exp_2023-10-23 21:14:50/models/model_best.pth.tar'
error: invalid path 'assignment2_release/logs/exp_2023-10-23 21:47:43/data_top1_accuracy_train/events.out.tfevents.1698116016.Mingyis-Book.17221.1'
error: invalid path 'assignment2_release/logs/exp_2023-10-23 21:47:43/data_top1_accuracy_val/events.out.tfevents.1698116040.Mingyis-Book.17221.3'
error: invalid path 'assignment2_release/logs/exp_2023-10-23 21:47:43/data_top5_accuracy_train/events.out.tfevents.1698116016.Mingyis-Book.17221.2'
error: invalid path 'assignment2_release/logs/exp_2023-10-23 21:47:43/data_top5_accuracy_val/events.out.tfevents.1698116040.Mingyis-Book.17221.4'
error: invalid path 'assignment2_release/logs/exp_2023-10-23 21:47:43/events.out.tfevents.1698115663.Mingyis-Book.17221.0'
error: invalid path 'assignment2_release/logs/exp_2023-10-23 21:47:43/models/checkpoint.pth.tar'
error: invalid path 'assignment2_release/logs/exp_2023-10-23 21:47:43/models/model_best.pth.tar'
``` 

You can either rename those files on GitHub using the browser editor, or you can bypass this protection by setting the following config:

```
git config --global core.protectNTFS false
```

You can view the reserved characters on Windows here:

http://msdn.microsoft.com/en-us/library/windows/desktop/aa365247(v=vs.85).aspx#naming_conventions

I hope that helps!

**Comment by steveward** [comment]
I also see some other errors in the log, so if that doesn't get things working let us know and we can investigate further!

**Comment by ysc12451** [comment]
Thanks for your help! Yes I agree that the problem is from the file names. But I have renamed all the files yesterday and can work normally in the command line. But the existing problem is that my github desktop still gets stuck in "fetching origin". I have tried removing the repo and adding it again, and reinstalling my github desktop. But either of them works. My github desktop still gets stuck in "fetching origin".
